pyproject.toml

```diff
@@ -112,9 +112,7 @@ module = [
     "onelogin.saml2.idp_metadata_parser.*",
     "openapi_core.*",
     "phabricator.*",
-    "phonenumbers.*",
     "progressbar.*",
-    "pytest_benchmark.*",
```

Comment:
this was actually removed in an earlier typing patch but nothing automatic caught it -- 🤷